.github/workflows/tests.yml

```
@@ -15,16 +15,20 @@ jobs:
       fail-fast: true
       matrix:
         os: [ ubuntu-latest ]
-        php: [ '8.0', '8.1', '8.2', '8.3' ]
-        laravel: [ '8.0', '9.33', '10.0', '11.0' ]
+        php: [ '8.0', '8.1', '8.2', '8.3', '8.4' ]
+        laravel: [ '8.0', '9.33', '10.0', '11.0', '12' ]
```

You've missed the `.0` part
```suggestion
        laravel: [ '8.0', '9.33', '10.0', '11.0', '12.0' ]
```